examples/d_dmrg.py

```diff
@@ -45,6 +55,43 @@ def example_DMRG_tf_ising_finite(L, g, verbose=True):
     return E, psi, M
 
 
+def example_1site_DMRG_tf_ising_finite(L, g, verbose=True):
```

Comment:
For that reason, and to avoid confusion about that parameter, I propose that we just make it more clear that we are using 1-site DMRG, by putting it in the printout string, as well as the docstring (the latter remains a to-do).